actions/telemetry_actions.jsx

```diff
@@ -182,3 +184,26 @@ export function trackPluginInitialization(plugins) {
         totalSize,
     });
 }
+
+export function trackSelectorMetrics() {
+    if (!shouldTrackPerformance()) {
+        return;
+    }
+
+    setTimeout(() => {
+        const selectors = getSortedTrackedSelectors();
+
+        trackEvent('performance', 'least_effective_selectors', {
+            after: 'one_minute',
+            first: selectors[0]?.name || '',
+            first_effectiveness: selectors[0]?.effectiveness,
+            first_recomputations: selectors[0]?.recomputations,
+            second: selectors[1]?.name || '',
+            second_effectiveness: selectors[1]?.effectiveness,
+            second_recomputations: selectors[1]?.recomputations,
+            third: selectors[2]?.name || '',
+            third_effectiveness: selectors[2]?.effectiveness,
+            third_recomputations: selectors[2]?.recomputations,
```

Comment:
@sri-byte is there a better way to report the "top 3 X things" with rudderstack? Or should this work ok?